aidants_connect_web/signals.py

```diff
@@ -38,7 +38,7 @@ def send_mail_aidant__organisations_changed(sender: Aidant, diff: dict, **_):
 
     send_mail(
         from_email=settings.AIDANTS__ORGANISATIONS_CHANGED_EMAIL_FROM,
-        recipient_list=[sender.email],
+        recipient_list=[instance.email],
```

Comment:
This is le fix !
Sender est la classe, or ici on avait besoin d'une instance de cette classe.